spec/vcr_cassettes/ManageIQ_Providers_CiscoIntersight_PhysicalInfraManager_Refresher/refresh/will_perform_a_full_refresh.yml

```diff
@@ -64244,4 +64244,362 @@ http_interactions:
         }
     http_version:
   recorded_at: Thu, 12 May 2022 13:47:31 GMT
+- request:
+    method: get
+    uri: https://intersight.com/api/v1/server/ProfilesTemplates
```

Comment:
```suggestion
    uri: https://intersight.com/api/v1/server/ProfileTemplates
```

Reply:
can you send me your local file to eilam.elimelech@kyndryl.com?